bindings/templates/modelEnum.mustache

```diff
@@ -0,0 +1,13 @@
+/**
+ * {{{description}}}
+ * @export
+ */
+export type {{classname}} = {{#allowableValues}}{{#enumVars}}{{{value}}}{{^@last}} | {{/@last}}{{/enumVars}}{{/allowableValues}};
```

Comment:
Can we make this the same as other enums in the codebase?
The generated code here looks like
```typescript
export type Determinedcontainerv1State = 'STATE_UNSPECIFIED' | 'STATE_ASSIGNED' | 'STATE_PULLING' | 'STATE_STARTING' | 'STATE_RUNNING' | 'STATE_TERMINATED';

export const Determinedcontainerv1State = {
    UNSPECIFIED: 'STATE_UNSPECIFIED' as Determinedcontainerv1State,
    ASSIGNED: 'STATE_ASSIGNED' as Determinedcontainerv1State,
    PULLING: 'STATE_PULLING' as Determinedcontainerv1State,
    STARTING: 'STATE_STARTING' as Determinedcontainerv1State,
    RUNNING: 'STATE_RUNNING' as Determinedcontainerv1State,
    TERMINATED: 'STATE_TERMINATED' as Determinedcontainerv1State
};
```
but we want
```typescript
export const Determinedcontainerv1State = {
    UNSPECIFIED: 'STATE_UNSPECIFIED',
    ASSIGNED: 'STATE_ASSIGNED',
    PULLING: 'STATE_PULLING',
    STARTING: 'STATE_STARTING',
    RUNNING: 'STATE_RUNNING',
    TERMINATED: 'STATE_TERMINATED',
} as const;
export type Determinedcontainerv1State = typeof Determinedcontainerv1State[keyof typeof Determinedcontainerv1State]
```
The latter allows easy subsetting of the enum.

Reply:
hmm this change affects code such as
```
export const isOfSortKey = (sortKey: React.Key): sortKey is V1GetExperimentTrialsRequestSortBy => {
  return Object.values(V1GetExperimentTrialsRequestSortBy).includes(sortKey);
};
```

will look into it